presto-hive/src/main/java/com/facebook/presto/hive/clustering/Distribution.java

```diff
@@ -0,0 +1,41 @@
+package com.facebook.presto.hive.clustering;
+
+import com.facebook.presto.hive.HiveType;
+import javafx.util.Pair;
```

Comment:
We don't use javafx usually. We can either create our own class or drop it in the body of the class.